test/parallel/test-console.js

```diff
@@ -118,6 +116,8 @@ for (const expected of expectedStrings) {
   assert.equal(expected + '\n', errStrings.shift());   // console.warn  (stderr)
 }
 
+assert.equal(expectedFuncToString, strings.shift());
+assert.equal(expectedFuncToString, strings.shift());
```

Comment:
There was some refactoring in the test case and my merge deleted some of the lines. This is just regaining the chakracore version of test case.